engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -209,7 +210,7 @@ IR.Module translateModule(Tree module) {
             var error = translateSyntaxError(inputAst, new IR$Error$Syntax$UnsupportedSyntax("Block without body"));
             yield cons(error, appendTo);
         }
-        var binding = new IR$Module$Scope$Definition$Method$Binding(
+        var binding = new org.enso.compiler.core.ir.module.scope.Definition$Method$Binding(
```

Comment:
Ops, this one is still pending... could it be renamed to `Definition.MethodBinding`?

Reply:
Yes, all of those will be gone. Just doing it one by one.

Reply:
In the next PR